analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -98,16 +98,23 @@ private fun String.extractOrganization(): String? =
         line.withoutPrefix(SpdxConstants.ORGANIZATION)
     }.firstOrNull()
 
+/**
+ * Return true if the id is that of an [SpdxExternalDocumentReference] if it has an additional package id attached to it
+ * or not. Otherwise return false.
+ */
+private fun String.isExternalDocumentReferenceId(): Boolean =
+    substringBefore(":").startsWith(SpdxConstants.DOCUMENT_REF_PREFIX)
+
 /**
  * Map a "not preset" SPDX value, i.e. NONE or NOASSERTION, to an empty string.
  */
 private fun String.mapNotPresentToEmpty(): String = takeUnless { SpdxConstants.isNotPresent(it) }.orEmpty()
 
 /**
- * Get the [SpdxPackage] from the [SpdxExternalDocumentReference], where [workingDir] is used to resolve local
- * relative URIs to files.
+ * Get the [SpdxPackage] from the [SpdxExternalDocumentReference] [packageId] refers to, where [workingDir] is used to
```

Comment:
Maybe insert a "that" between "[SpdxExternalDocumentReference]" and "[packageId]" to have those two linked words separated by a plain word that is *not* linked, so "[SpdxExternalDocumentReference] [packageId]" does not look like one link in rendered docs?